boards/shields/nrf7002_ek/Kconfig.shield

```diff
@@ -4,5 +4,14 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
This directory has to be removed as we are using the new directory

Reply:
Done